src/mono/wasm/runtime/es6/dotnet.es6.lib.js

```diff
@@ -121,8 +120,7 @@ const linked_functions = [
 // -- this javascript file is evaluated by emcc during compilation! --
 // we generate simple proxy for each exported function so that emcc will include them in the final output
 for (let linked_function of linked_functions) {
-    const fn_template = `return __dotnet_runtime.__linker_exports.${linked_function}.apply(__dotnet_runtime, arguments)`;
-    DotnetSupportLib[linked_function] = new Function(fn_template);
+    DotnetSupportLib[linked_function] = new Function("");
```

Comment:
This change breaks calling into mono C functions on threads because `replace_linker_placeholders` is not installed there. 

cc @kg @lambdageek @thaystg

Reply:
I opened https://github.com/dotnet/runtime/issues/81268 for it